spec/unit/pundit_adapter_spec.rb

```diff
@@ -91,6 +91,34 @@ def resolve
       auth.authorized?(:index)
     end
 
+    context "when model name contains policy namespace name" do
+      before do
+        allow(ActiveAdmin.application).to receive(:pundit_policy_namespace).and_return :pub
+        namespace.register(Publisher)
+      end
+
+      around do |example|
+        original_stderr = $stderr
+        $stderr = StringIO.new
+        example.run
+        $stderr = original_stderr
```

Comment:
This trick is also used at `spec/unit/action_builder_spec.rb`. Can we extract it to a helper, like `capturing_stderr(&example)` or something?